journal/src/main/resources/journal-record-schema.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<sbe:messageSchema xmlns:sbe="http://fixprotocol.io/2016/sbe"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  package="io.zeebe.journal.file" id="7" version="1"
+  semanticVersion="0.1.0" description="Zeebe Journal Record Protocol" byteOrder="littleEndian">
```

Comment:
We need to do something similar to keep the versions in line as we did in the `zeebe-protocol` I guess. Can be done as a follow up.